kafka/handle.go

```diff
@@ -28,9 +28,50 @@ import (
 */
 import "C"
 
+// OAuthBearerToken represents the data to be transmitted
+// to a broker during SASL/OAUTHBEARER authentication.
+type OAuthBearerToken struct {
+	// Token value, often (but not necessarily) a JWS compact serialization
+	// as per https://tools.ietf.org/html/rfc7515#section-3.1; it must meet
+	// the regular expression for a SASL/OAUTHBEARER value defined at
+	// https://tools.ietf.org/html/rfc7628#section-3.1
+	TokenValue string
+	// Metadata about the token indicating when it expires
+	// in terms of the number of milliseconds since the epoch; it must
+	// represent a time in the future
+	LifetimeMilliseconds int64
```

Comment:
Shouldn't we use the Time type here?
Is there any value/correctness in using int64 instead?

Reply:
👍